xlsxwriter.class.php

```diff
@@ -258,9 +264,9 @@ public function writeSheetRow($sheet_name, array $row, $row_options=null)
 		if (!empty($row_options))
 		{
 			$ht = isset($row_options['height']) ? floatval($row_options['height']) : 12.1;
-			$customHt = isset($row_options['height']) ? true : false;
-			$hidden = isset($row_options['hidden']) ? (bool)($row_options['hidden']) : false;
-			$collapsed = isset($row_options['collapsed']) ? (bool)($row_options['collapsed']) : false;
+			$customHt = isset($row_options['height']) ? 'true' : 'false';
+			$hidden = isset($row_options['hidden']) ? (bool)($row_options['hidden']) : 'false';
+			$collapsed = isset($row_options['collapsed']) ? (bool)($row_options['collapsed']) : 'false';
```

Comment:
this still does not use the right value when passing the options (as they are still boolean there rather than strings)